src/main/kotlin/com/github/devcordde/devcordbot/command/context/Context.kt

```diff
@@ -151,7 +153,7 @@ data class Context(
         commandClient.permissionHandler.isCovered(permission, member, devCordUser) == PermissionState.ACCEPTED
 
     private fun notifyCommandHandler(action: MessageAction) = action.map {
-        (commandClient as CommandClientImpl).acknowledgeResponse(message.idLong, it.channel.idLong, it.idLong); it
+//        (commandClient as CommandClientImpl).acknowledgeResponse(message.idLong, it.channel.idLong, it.idLong); it
```

Comment:
Please specify why this is commented out

Reply:
No longer needed as slash commands store responses and follow ups by themselves

Reply:
Why do we keep it then? Also, I meant an explanation in the code

Reply:
because i forgot to remove it